### ISSUE TITLE
PyYAML warning with the use of yaml.load

### ISSUE THREAD

**Issue by fgimian** [circus-tent/circus#1144]
Hey there guys, absolutely love circus!

I noticed a little warning when starting my app and using a YAML logging config:

```
/home/fots/.virtualenv/testing/lib/python3.8/site-packages/circus/util.py:741: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  logging.config.dictConfig(yaml.load(fh.read()))
```

My PyYAML version is 5.3.1.

I believe it's best to use `yaml.safe_load` which is more secure and will remove this warning.

Cheers
Fotis

**Comment by biozz** [comment]
Hi @fgimian!

Thank you for pointing that out.

These warnings have been fixed in #1126, but not released yet. The next release date is yet to be decided, you can track it here #1130. 

There was also an attempt for a workaround in #1118 and this might be the option for now to just pin PyYAML version below 5.3 in your requirements file.

**Comment by fgimian** [comment]
Thank you so much @biozz 😄
